services/server/src/server/apiv1/verification/private/stateless/private.stateless.handlers.ts

```diff
@@ -134,6 +134,13 @@ export async function verifyDeprecated(
   }
 }
 
+/**
+ * Upgrades a contract by fetching the sourcify_match from the database and then verifying it again with the new verification parameters.
```

Comment:
> Upgrades a contract...

by creating a new `verified_contracts` entry or modifies the existing entry?

> with the new verification parameters.

What are the new verification parameters?

Reply:
What about this?

```
/**
* This function is used to fix the creation information (match, transformations list, transformations values, metadata match) of contract with misaligned creation data.
 * This function mocks both the compilation and the sourcifyChain information so that recompilation and rpc calls are not performed. The mocked information is fetched from the database.
 * Finally it upgrades a contract by UPDATING the verified_contracts and sourcify_matches creation fields.
 */
```

Reply:
I updated the comment, let me know if not it's more clear.